Services/BetQuoteServices.cs

```diff
@@ -7,13 +7,15 @@
 
 namespace Services
 {
-    public class BetQuoteServices : IBetableService<BetQuotes, UpdateBetQuotes>
+    public sealed class BetQuoteServices : IBetQuoteService
```

Comment:
```suggestion
    public sealed class BetQuoteService : IBetQuoteService
```
let's keep it in singular